appium-examples/src/test/java/com/emusim/biometric_login/BiometricLoginIosTest.java

```diff
@@ -114,15 +124,32 @@ public void biometricLoginWithNonMatchingTouch () throws InterruptedException {
         this.login(false);
 
         // Verificsation
-        Assert.assertTrue(this.isRetryBiometryDisplay(Boolean.valueOf(rdc)));
+        Assertions.assertTrue(this.isRetryBiometryDisplay(Boolean.valueOf(rdc)));
 
     }
 
-    @AfterMethod
-    public void teardown(ITestResult result) {
-        System.out.println("Sauce - AfterMethod hook");
-        ((JavascriptExecutor)driver).executeScript("sauce:job-result=" + (result.isSuccess() ? "passed" : "failed"));
-        driver.quit();
+    public class MyTestWatcher implements TestWatcher {
+        @Override
+        public void testSuccessful(ExtensionContext context) {
+            try {
+                System.out.println("Test Passed!");
```

Comment:
@eyaly maybe we should be using a Log statement here instead of a System.out as that can be enabled or disabled